pkg/network/netlink/conntrack_integration_test.go

```diff
@@ -87,7 +92,7 @@ func testConntrackExists(t *testing.T, laddrIP string, laddrPort int, proto stri
 				},
 			},
 			exists: true,
-			ns:     0,
+			ns:     int(rootNs),
```

Comment:
If I'm reading this correctly,  `NewConntrack(0)` connects to the root namespace  on all systems but debian 10 -- why is that ?

Reply:
The problem is not with the code, it is with system conntrack. Apparently conntrack is only enabled if there is at least one iptables rules that uses it (NAT, etc.). For root ns, in this test, we don't have a rule like that on debian in this case, but it can happen on other distros too.